tests/features/tokens/generate.feature

```diff
@@ -0,0 +1,70 @@
+Feature: Generate HTTP Token for Repository Service for TUF (RSTUF)
+    As an admin,
+    Admin has deployed and bootstrapped RSTUF
+
++
+    Scenario Outline: Admin uses HTTP API to generate Token
+        Given the admin has the admin password
+        And the admin has generated the admin "access_token" after login "/api/v1/token"
```

Comment:
From what I understand you can generate more than one token of this kind, right?
There could be more than one, correct?

Also, you don't specify the scope of this access token which is important to be able to generate a new
access token you need to have the `write:token` scope.

Finally, I don't think mentioning that this is `an admin access_token` is important (as this is a regular access token) or mentioning which method you use to generate that access token (this is unnecessary detail)

Here is my suggestion on how to improve:
```suggestion
        And the admin has generated an "access_token" after login with a "write:token" scope
```

Reply:
> From what I understand you can generate more than one token of this kind, right? There could be more than one, correct?

The admin can generate multiple tokens, but one per request. Not clear if I got your point here.

> Also, you don't specify the scope of this access token which is important to be able to generate a new access token you need to have the `write:token` scope.

👍 

> Finally, I don't think mentioning that this is `an admin access_token` is important (as this is a regular access token) or mentioning which method you use to generate that access token (this is unnecessary detail)

It is totally important. If the admin login fails or "access_token" isn't generated in the response, the admin cannot proceed to the next "Given" (note: all "And" are another "Given")